### PR TITLE
ko: cr2 Day 4 translations

### PR DESCRIPTION
**Pull request by henrif75**
#262 #1463
*Stats*
Before: 1297 translated messages, 383 fuzzy translations, 1780 untranslated messages.
After: 1492 translated messages, 516 fuzzy translations, 1452 untranslated messages.

### CODE REVIEW

**Comment by henrif75** [comment]
Stats after clean-up:
1490 translated messages, 396 fuzzy translations, 1574 untranslated messages.

**Comment by namhyung** [comment]
Please convert incorrect `'` to `\"` in the code strings.

File: po/ko.po [hunk 1]
```diff
@@ -9028,6 +9027,7 @@ msgstr "오류처리(에러 핸들링): 패닉, `Result`, `?` 연산자."
 msgid ""
 "Unsafe Rust: the escape hatch when you can't express yourself in safe Rust."
 msgstr ""
+"안전하지 않은 Rust: 안전한 Rust로 표현할 수 없을 때의 이스케이프 해치입니다."
```
Comment (namhyung) [inline po/ko.po:9030]:
```suggestion
"안전하지 않은 Rust: 안전한 Rust로 표현할 수 없을 때의 비상 수단입니다."
```

File: po/ko.po [hunk 2]
```diff
@@ -9080,7 +9084,7 @@ msgstr ""
 
 #: src/iterators/intoiterator.md:49
 msgid "\"point = {x}, {y}\""
-msgstr ""
+msgstr "\"포인트 = {x}, {y}\""
```
Comment (namhyung) [inline po/ko.po:9087]:
```suggestion
msgstr "\"point = {x}, {y}\""
```

File: po/ko.po [hunk 3]
```diff
@@ -9492,12 +9531,17 @@ msgid ""
 "It is typical to put each type or set of closely-related types into its own "
 "module, so each widget type should get its own module."
 msgstr ""
+"이 연습에서는 과정의 '메서드 및 트레잇' 세그먼트의 GUI 라이브러리 연습을 모"
```
Comment (namhyung) [inline po/ko.po:9534]:
```suggestion
"이 연습에서는 '메서드 및 트레잇' 세그먼트의 GUI 라이브러리 연습을 모"
```

File: po/ko.po [hunk 4]
```diff
@@ -9492,12 +9531,17 @@ msgid ""
...
+"듈 컬렉션으로 재구성합니다. 각 타입 또는 밀접하게 관련된 타입 집합을 자체 모"
+"듈에 배치하는 것이 일반적이므로 각 위젯 타입은 자체 모듈을 가져야 합니다."
 
 #: src/modules/exercise.md:8
 msgid ""
 "If you no longer have your version, that's fine - refer back to the "
 "[provided solution](../methods-and-traits/solution.html)."
 msgstr ""
+"더 이상 버전이 없어도 괜찮습니다. [제공된 솔루션](../methods-and-traits/"
```
Comment (namhyung) [inline po/ko.po:9543]:
```suggestion
"예전에 작업했던 버전이 없어도 괜찮습니다. [제공된 솔루션](../methods-and-traits/"
```

File: po/ko.po [hunk 5]
```diff
@@ -9509,77 +9553,85 @@ msgid ""
...
+"다."
 
 #: src/modules/exercise.md:27
 msgid ""
 "Encourage students to divide the code in a way that feels natural for them, "
 "and get accustomed to the required `mod`, `use`, and `pub` declarations. "
 "Afterward, discuss what organizations are most idiomatic."
 msgstr ""
+"학생들에게 자연스러운 방식으로 코드를 나누고 필수 `mod`, `use`, `pub` 선언에 "
+"익숙해지도록 권장합니다. 그런 다음 어떤 구성이 가장 자연스러운지 논의합니다."
```
Comment (namhyung) [inline po/ko.po:9574]:
```suggestion
"학생들에게는 본인에게 자연스러운 방식으로 코드를 나누도록 권장하세요. "
"그리고나서 필요한 `mod`, `use`, `pub` 선언에 익숙해지도록 합니다. "
"그런 다음 어떤 구성이 가장 자연스러운지 논의합니다."
```

**Review by namhyung — COMMENTED** [review]
Looks like the module exercise is outdated.

File: po/ko.po [hunk 6]
```diff
@@ -9739,76 +9796,95 @@ msgstr ""
...
 
 #: src/testing/googletest.md:51
 msgid ""
 "\"Memory safety found,\\n\\\n"
 "                 Rust's strong typing guides the way,\\n\\\n"
 "                 Secure code you'll write.\""
 msgstr ""
+"\"메모리 안전을 확보했습니다.\\n\\\n"
+" Rust의 강력한 타입이 길을 안내합니다.\\n\\\n"
+" 작성할 코드를 안전하게 보호하세요.\""
```
Comment (namhyung) [inline po/ko.po:9851]:
```suggestion
"\"Memory safete found,\\n\\\n"
"                 Rust's strong typing guides the way,\\n\\\n"
"                 Secure code you'll write.\""
```

File: po/ko.po [hunk 7]
```diff
@@ -9739,76 +9796,95 @@ msgstr ""
...
 
 #: src/testing/googletest.md:56
 msgid ""
 "\"Memory safety found,\\n\\\n"
 "            Rust's silly humor guides the way,\\n\\\n"
 "            Secure code you'll write.\""
 msgstr ""
+"\"메모리 안전을 확보했습니다.\\n\\\n"
+" Rust의 우스꽝스러운 유머가 길을 안내합니다.\\n\\\n"
+" 작성할 코드를 안전하게 보호하세요.\""
```
Comment (namhyung) [inline po/ko.po:9861]:
```suggestion
"\"Memory safety found,\\n\\\n"
"            Rust's silly humor guides the way,\\n\\\n"
"            Secure code you'll write.\""
```

File: po/ko.po [hunk 8]
```diff
@@ -9739,76 +9796,95 @@ msgstr ""
...
 
 #: src/testing/googletest.md:63
 msgid "shows a color-coded diff (colors not shown here):"
-msgstr ""
+msgstr "색상으로 구분된 diff를 표시합니다(여기에 표시되지 않은 색상)."
```
Comment (namhyung) [inline po/ko.po:9865]:
```suggestion
msgstr "색상으로 구분된 diff를 표시합니다(여기에서는 색상이 표시되지 않습니다)."
```

File: po/ko.po [hunk 9]
```diff
@@ -9009,11 +9008,11 @@ msgstr "오늘은 몇 가지 고급 주제를 다룹니다:"
 
 #: src/welcome-day-4.md:5
 msgid "Iterators: a deep dive on the `Iterator` trait."
-msgstr ""
+msgstr "반복자: `Iterator` 트레잇 심층 분석"
 
 #: src/welcome-day-4.md:6
 msgid "Modules and visibility."
-msgstr ""
+msgstr "모듈 및 공개 상태"
```
Comment (jiyongp) [inline po/ko.po:9015]:
```suggestion
msgstr "모듈과 가시성"
```

File: po/ko.po (hunk 1 above)
Comment (jiyongp) [inline po/ko.po:9030]:
```suggestion
"안전하지 않은 Rust: 안전한 Rust로 원하는 것을 표현할 수 없을 때에만 사용하세요."
```

File: po/ko.po [hunk 10]
```diff
@@ -9036,10 +9036,14 @@ msgid ""
 "method and provides lots of methods. Many standard library types implement "
 "`Iterator`, and you can implement it yourself, too:"
 msgstr ""
+"[`Iterator`](https://doc.rust-lang.org/std/iter/trait.Iterator.html) 트레잇"
+"은 컬렉션의 값에 대한 반복을 지원합니다. `next` 메서드가 필요하며 많은 메서드"
+"를 제공합니다. 많은 표준 라이브러리 타입에서 `Iterator`를 구현하며, 개발자가 "
+"직접 구현할 수도 있습니다."
```
Comment (jiyongp) [inline po/ko.po:9042]:
```suggestion
"컬랙션에 있는 값들을 접근하기 위해서는 [`Iterator`](https://doc.rust-lang.org/std/iter/trait.Iterator.html) 트레잇"
"을 사용합니다. 이 트레잇은 `next` 메서드를 비롯한 많은 메서드"
"를 제공합니다. 많은 표준 라이브러리 타입이 `Iterator`를 구현하고 있으며, 여러분도 여러분의 타입이 "
"이 트레잇을 직접 구현하도록 할 수 있습니다."
```

File: po/ko.po [hunk 11]
```diff
@@ -9109,19 +9113,23 @@ msgstr ""
 
 #: src/iterators/intoiterator.md:65
 msgid "The example iterates over all combinations of x and y coordinates."
-msgstr ""
+msgstr "이 예는 x 및 y 좌표의 모든 조합을 반복합니다."
```
Comment (jiyongp) [inline po/ko.po:9116]:
```suggestion
msgstr "이 예는 x 및 y 좌표의 모든 조합을 순회합니다."
```

File: po/ko.po [hunk 12]
```diff
@@ -9156,20 +9168,26 @@ msgstr "`Iterator`"
...
 #: src/iterators/fromiterator.md:26
+#, fuzzy
 msgid ""
 "With the \"turbofish\": `some_iterator.collect::<COLLECTION_TYPE>()`, as "
 "shown. The `_` shorthand used here lets Rust infer the type of the `Vec` "
 "elements."
 msgstr ""
+"\"turbofish' 사용: 다음과 같이 `some_iterator.collect::<COLLECTION_TYPE>()`"
+"을 사용합니다. 여기에 사용된 `_` 약식을 통해 Rust가 `Vec` 요소의 타입을 추론"
+"할 수 있습니다. "
```
Comment (jiyongp) [inline po/ko.po:9182]:
```suggestion
"\"turbofish' 사용: 다음과 같이 `some_iterator.collect::<COLLECTION_TYPE>()`"
"을 사용합니다. 여기에 사용된 `_` 는 Rust가 `Vec` 에 들어갈 요소들의 타입을 추론하도록"
"합니다. "
```

File: po/ko.po [hunk 13]
```diff
@@ -9156,20 +9168,26 @@ msgstr "`Iterator`"
...
 
 #: src/iterators/fromiterator.md:28
 msgid ""
 "With type inference: `let prime_squares: Vec<_> = some_iterator.collect()`. "
 "Rewrite the example to use this form."
 msgstr ""
+"타입 추론 사용: `let prime_squares: Vec<_> = some_iterator.collect()`를 사용"
+"합니다. 이 양식을 사용하려면 예를 다시 작성하세요."
```
Comment (jiyongp) [inline po/ko.po:9190]:
```suggestion
"타입 추론 사용: `let prime_squares: Vec<_> = some_iterator.collect()`를 사용"
"합니다. 이 방법으로 예제를 다시 작성해 보세요."
```

File: po/ko.po [hunk 14]
```diff
@@ -9287,18 +9311,20 @@ msgid ""
 "germination\n"
 "//! implementation.\n"
 msgstr ""
+"//! 이 모듈은 높은 성능의 발아 구현을 비롯하여 정원을\n"
+"//! 구현합니다.\n"
```
Comment (jiyongp) [inline po/ko.po:9315]:
```suggestion
"//! 이 모듈은 정원을 구현합니다.\n"
```

**Review by jiyongp — COMMENTED** [review]
I've made some inline suggestions.

**Review by jiyongp — APPROVED** [review]

**Comment by henrif75** [comment]
Thank you!

**Pull request merged**